android/src/main/java/com/google/samples/apps/iosched/droidconitaly15/JsonTransformer.java

```diff
@@ -223,23 +245,98 @@ private static void transformJson(
+
+        addBlock("Welcome", "Sala 500", day, "09:15", "09:50", false,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("Barcamp organization", "Sala 500", day, "10:30", "10:50", false,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("", "", day, "11:00", "12:00", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "12:00", "12:50", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "12:50", "13:30", true,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("Lunch", "", day, "13:30", "14:30", false,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("", "", day, "14:30", "15:10", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "15:10", "16:10", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "16:10", "17:10", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "17:10", "18:00", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "18:00", "19:00", true,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("Droidcon Party!", "Caffè del Progresso", day, "19:00", "23:00", false,
+                dateParser, dateFormatterBlocks, blocks);
+
+        day = "10 April 2015";
+
+        addBlock("", "", day, "09:00", "09:50", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "09:50", "10:30", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "10:30", "11:40", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "11:40", "12:20", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "12:20", "13:00", true,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("Lunch", "", day, "13:00", "14:00", false,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("", "", day, "14:00", "14:50", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "14:50", "15:40", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "15:40", "16:40", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "16:40", "17:30", true,
+                dateParser, dateFormatterBlocks, blocks);
+        addBlock("", "", day, "17:30", "18:20", true,
+                dateParser, dateFormatterBlocks, blocks);
+
+        addBlock("Closing ceremony", "Sala Londra", day, "18:20", "18:40", false,
+                dateParser, dateFormatterBlocks, blocks);
+    }
+
+    private static void addBlock(
+            @NonNull String title,
```

Comment:
Ah dude you even put the effort of annotating the params! Nothing else in the codebase does lol